src/window.ts

```diff
@@ -0,0 +1,11 @@
+import type { BrowserWindow } from 'electron';
+
+export const showWindowOnTop = (window: BrowserWindow) => {
+  window.setAlwaysOnTop(true);
+  // Hack to show on current desktop - see: https://github.com/electron/electron/issues/5362
+  window.setVisibleOnAllWorkspaces(true, { visibleOnFullScreen: true });
+  window.show();
+  window.focus();
+  window.setVisibleOnAllWorkspaces(false);
+  window.setAlwaysOnTop(false);
+};
```

Comment:
What's the reason for this being a separate file?

Reply:
Just for reusability. It is used in `main.ts` and the saga.

Reply:
Can you move it to `utils` then?

Reply:
Sure